apps/framework-landing/app/sections/home/HostWithBorealSection.tsx

```diff
@@ -9,7 +9,7 @@ import React from "react";
 
 export const SecondaryCTASection = () => {
   const content = {
-    title: "Host your Moose projects with Bóreal",
+    title: "Host your Moose projects with Boréal",
```

Comment:
One of the nav links is "Host with Boreal", do we want to add the accent to that as well?